`google-cloud-audit-log >= 0.1.1` is needed to resolve this error which appears with `google-cloud-audit-log == 0.1.0`

```
______ ERROR collecting tests/unit/handlers/test__monitored_resources.py _______
tests/unit/handlers/test__monitored_resources.py:23: in 
    from google.cloud.logging_v2.handlers._monitored_resources import (
google/cloud/logging_v2/__init__.py:21: in 
    from google.cloud.logging_v2.client import Client
google/cloud/logging_v2/client.py:25: in 
    from google.cloud.logging_v2._helpers import _add_defaults_to_filter
google/cloud/logging_v2/_helpers.py:25: in 
    from google.cloud.logging_v2.entries import LogEntry
google/cloud/logging_v2/entries.py:31: in 
    import google.cloud.audit.audit_log_pb2  # noqa: F401
.nox/unit-3-7-protobuf_implementation-python/lib/python3.7/site-packages/google/cloud/audit/audit_log_pb2.py:48: in 
    google_dot_rpc_dot_status__pb2.DESCRIPTOR,
.nox/unit-3-7-protobuf_implementation-python/lib/python3.7/site-packages/google/protobuf/descriptor.py:1034: in __init__
    _Deprecated('FileDescriptor')
.nox/unit-3-7-protobuf_implementation-python/lib/python3.7/site-packages/google/protobuf/descriptor.py:102: in _Deprecated
    category=DeprecationWarning, stacklevel=3)
E   DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
```

--- a/setup.py
+++ b/setup.py
@@ -44,7 +44,7 @@ dependencies = [
     "google-cloud-audit-log >= 0.1.1, < 1.0.0dev",
     "google-cloud-core >= 2.0.0, <3.0.0dev",
     "grpc-google-iam-v1 >=0.12.4, <1.0.0dev",
-    "opentelemetry-api >= 1.0.0",
+    "opentelemetry-api >= 1.9.0",
     "proto-plus >= 1.22.0, <2.0.0dev",
     "proto-plus >= 1.22.2, <2.0.0dev; python_version>='3.11'",
     "protobuf>=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",

--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,7 @@ dependencies = [
     # See https://github.com/googleapis/google-cloud-python/issues/12364
     "google-auth >= 2.14.1, <3.0.0dev,!=2.24.0,!=2.25.0",
     "google-cloud-appengine-logging>=0.1.3, <2.0.0dev",
-    "google-cloud-audit-log >= 0.1.1, < 1.0.0dev",
+    "google-cloud-audit-log >= 0.2.4, < 1.0.0dev",
     "google-cloud-core >= 2.0.0, <3.0.0dev",
     "grpc-google-iam-v1 >=0.12.4, <1.0.0dev",
     "opentelemetry-api >= 1.9.0",

--- a/setup.py
+++ b/setup.py
@@ -40,7 +40,7 @@ dependencies = [
     # Exclude incompatible versions of `google-auth`
     # See https://github.com/googleapis/google-cloud-python/issues/12364
     "google-auth >= 2.14.1, <3.0.0dev,!=2.24.0,!=2.25.0",
-    "google-cloud-appengine-logging>=0.1.0, <2.0.0dev",
+    "google-cloud-appengine-logging>=0.1.3, <2.0.0dev",
     "google-cloud-audit-log >= 0.1.1, < 1.0.0dev",
     "google-cloud-core >= 2.0.0, <3.0.0dev",
     "grpc-google-iam-v1 >=0.12.4, <1.0.0dev",

--- a/testing/constraints-3.7.txt
+++ b/testing/constraints-3.7.txt
@@ -10,7 +10,7 @@ proto-plus==1.22.0
 protobuf==3.20.2
 google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.3
-google-cloud-audit-log==0.1.1
+google-cloud-audit-log==0.2.4
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0
 

--- a/testing/constraints-3.7.txt
+++ b/testing/constraints-3.7.txt
@@ -9,7 +9,7 @@ google-auth==2.14.1
 proto-plus==1.22.0
 protobuf==3.20.2
 google-cloud-core==2.0.0
-google-cloud-appengine-logging==0.1.0
+google-cloud-appengine-logging==0.1.3
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0

--- a/testing/constraints-3.7.txt
+++ b/testing/constraints-3.7.txt
@@ -12,7 +12,7 @@ google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.0
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
-opentelemetry-api==1.0.0
+opentelemetry-api==1.9.0
 
 # Lower bound testing for optional dependencies
 django==3.2

--- a/testing/constraints-3.8.txt
+++ b/testing/constraints-3.8.txt
@@ -6,7 +6,7 @@ google-auth==2.14.1
 proto-plus==1.22.0
 protobuf==4.21.6
 google-cloud-core==2.0.0
-google-cloud-appengine-logging==0.1.0
+google-cloud-appengine-logging==0.1.3
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0

--- a/testing/constraints-3.8.txt
+++ b/testing/constraints-3.8.txt
@@ -9,7 +9,7 @@ google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.0
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
-opentelemetry-api==1.0.0
+opentelemetry-api==1.9.0
 
 # Lower bound testing for optional dependencies
 django==3.2

--- a/testing/constraints-3.8.txt
+++ b/testing/constraints-3.8.txt
@@ -7,7 +7,7 @@ proto-plus==1.22.0
 protobuf==4.21.6
 google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.3
-google-cloud-audit-log==0.1.1
+google-cloud-audit-log==0.2.4
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0
 

--- a/testing/constraints-3.9.txt
+++ b/testing/constraints-3.9.txt
@@ -7,7 +7,7 @@ proto-plus==1.22.0
 protobuf==5.26.0
 google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.3
-google-cloud-audit-log==0.1.1
+google-cloud-audit-log==0.2.4
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0
 

--- a/testing/constraints-3.9.txt
+++ b/testing/constraints-3.9.txt
@@ -6,7 +6,7 @@ google-auth==2.14.1
 proto-plus==1.22.0
 protobuf==5.26.0
 google-cloud-core==2.0.0
-google-cloud-appengine-logging==0.1.0
+google-cloud-appengine-logging==0.1.3
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
 opentelemetry-api==1.9.0

--- a/testing/constraints-3.9.txt
+++ b/testing/constraints-3.9.txt
@@ -9,7 +9,7 @@ google-cloud-core==2.0.0
 google-cloud-appengine-logging==0.1.0
 google-cloud-audit-log==0.1.1
 grpc-google-iam-v1==0.12.4
-opentelemetry-api==1.0.0
+opentelemetry-api==1.9.0
 
 # Lower bound testing for optional dependencies
 django==3.2

